examples/svm/plot_svm_anova.py

```diff
@@ -20,14 +20,14 @@
 from sklearn.svm import SVC
 
 
-# #############################################################################
+# %%
 # Import some data to play with
 X, y = load_iris(return_X_y=True)
 # Add non-informative features
 np.random.seed(0)
 X = np.hstack((X, 2 * np.random.random((X.shape[0], 36))))
```

Comment:
Nowadays we tend to use the rng API explicitly to avoid writing code with side effects on the shared global RNG singleton of numpy.

```suggestion
rng = np.random.RandomState(0)
X = np.hstack((X, 2 * rng.random((X.shape[0], 36))))
```